.github/workflows/test-install.yml

```diff
@@ -43,9 +43,11 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Set up Python 3.7
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
```

Comment:
Since here we do not necessarily need Python 3.7 but just need _a_ version, why not switch to Python 3.8? We are anyway testing Python 3.7 in the `tests` job where it really matters.

Fine to keep the updated version of the `setup-python` action, but I would update it everywhere in all workflows.

Reply:
Done. 👍

Reply:
Done. 👍